ext/pdo_sqlite/sqlite_driver.c

```diff
@@ -309,12 +309,12 @@ typedef struct {
 	zend_long row;
 } aggregate_context;
 
-static int do_callback(struct pdo_sqlite_fci *fc, zval *cb, int argc, sqlite3_value **argv, sqlite3_context *context, int is_agg)
+static int do_callback(zend_fcall_info_cache *fcc, int argc, sqlite3_value **argv, sqlite3_context *context, int is_agg)
 {
 	zval *zargs = NULL;
 	zval retval;
 	int i;
-	int ret;
+	int ret = SUCCESS;
```

Comment:
Either this function should return a `zend_result` or use integers.

It also looks like we don't even use the return value when calling this function.

Reply:
Agree. However, there are probably some other similar ones, so I will respond with a follow-up PR.